xchange-stream-bitmex/src/test/java/info/bitrich/xchangestream/bitmex/BitmexDeadManSwitchIT.java

```diff
@@ -4,7 +4,7 @@
 
 import info.bitrich.xchangestream.core.StreamingExchange;
 import java.math.BigDecimal;
-import org.junit.Ignore;
+
```

Comment:
Any chance of changing all those `...Example` classes which use `main` methods to use this pattern too?  (Feel free to say "some other time" - it would just be nice to do it all at once).